src/main/java/hudson/plugins/git/extensions/impl/CloneOption.java

```diff
@@ -148,7 +149,20 @@ public void decorateCloneCommand(GitSCM scm, Run<?, ?> build, GitClient git, Tas
             // in a single job definition.
             RemoteConfig rc = scm.getRepositories().get(0);
             List<RefSpec> refspecs = rc.getFetchRefSpecs();
-            cmd.refspecs(refspecs);
+            List<RefSpec> expandedrefSpecs = new ArrayList<>();
+            Boolean check = false;
+            for (RefSpec ref:refspecs) {
+                if(ref.toString().contains("$")){
+                    EnvVars env = build.getEnvironment(listener);
+                    expandedrefSpecs.add(new RefSpec(env.expand(ref.toString())));
+                    check = true;
+                }
+            }
+            if(!check) {
+                cmd.refspecs(refspecs);
+            } else{
```

Comment:
I'm not sure if git-pugin has coding style, but the spacing for braces is inconsistent.